packages/walletconnect/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.json",
   "include": ["./src"],
   "compilerOptions": {
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "esModuleInterop": true
```

Comment:
Typescript complained this was required for the `eventemitter3`. This is interesting, because pre and post my work, the version is the same. `tsc` is actually not doing type checking while compiling, and I guess that's the reason why it worked. Now that all the remaining type errors are gone, I have updated this configuration to make it have no errors at all.